cmd/explaintest/r/tpch.result

```diff
@@ -769,14 +769,16 @@ l_shipmode;
 id	count	task	operator info
 Sort_9	1.00	root	tpch.lineitem.l_shipmode:asc
 └─Projection_11	1.00	root	tpch.lineitem.l_shipmode, Column#27, Column#28
-  └─HashAgg_14	1.00	root	group by:Column#35, funcs:sum(Column#32)->Column#27, funcs:sum(Column#33)->Column#28, funcs:firstrow(Column#34)->tpch.lineitem.l_shipmode
-    └─Projection_40	10023369.01	root	cast(case(or(eq(tpch.orders.o_orderpriority, 1-URGENT), eq(tpch.orders.o_orderpriority, 2-HIGH)), 1, 0))->Column#32, cast(case(and(ne(tpch.orders.o_orderpriority, 1-URGENT), ne(tpch.orders.o_orderpriority, 2-HIGH)), 1, 0))->Column#33, tpch.lineitem.l_shipmode, tpch.lineitem.l_shipmode
-      └─IndexMergeJoin_22	10023369.01	root	inner join, inner:TableReader_20, outer key:tpch.lineitem.l_orderkey, inner key:tpch.orders.o_orderkey
```

Comment:
It seems the outer child and inner child are swapped. Is it expected?

Reply:
Because I removed the logic of choosing smaller table as IndexJoin outer child in `tryToGetIndexJoin`. It was reasonable in old cost model since we only considered outer row count in cost computing of IndexJoin then, but this logic is out of date for the current cost model.

Reply:
The plan structure is not changed now.